content/concepts/transport.md

```diff
@@ -115,3 +115,190 @@ and some places in the codebase still use the "swarm" terminology.
 {{% /notice %}}
 
 [definition_switch]: /reference/glossary/#switch
+
+## QUIC
+
+Transport protocols continue to improve transport methods in an attempt to alleviate 
+the shortcomings of the transport layer. 
```

Comment:
I don’t understand this sentence. How about a quick summary of what QUIC is instead, something like “QUIC is a new transport protocol that provides an always-encrypted, stream-multiplexed connection built on top of UDP. It started as an experiment by Google in xxxx and was later standardized by the IETF <link to RFC 9000>”?

Reply:
Updated - combined the later paragraph with similar content which now introduces the section